specification/ai/Azure.AI.Agents/tools/models.tsp

```diff
@@ -64,6 +64,7 @@ model BingGroundingToolDefinition extends ToolDefinition {
 @doc("The fabric data agent tool parameters.")
 @added(Versions.v2025_05_15_preview)
 @removed(Versions.v1)
+@removed(Versions.v2025_05_01)
```

Comment:
This doesn't make sense - they were already removed in Versions.v1. Something is wrong here.

Reply:
Removed the @Remove decorators.